### DISCUSSION TITLE
Browser tab notifications

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3972] [Q&A] [answered]
Hi Paste team,

We are working on a functionality where a user can see a tab notification for new chat messages if they go to a different browser screen.  See example below:
![341794503-0a7000a9-2f92-4dc5-9d36-6fd15bcee40a](https://github.com/twilio-labs/paste/assets/9394413/6b364e96-f6d3-4086-bebb-ada3c4004f2c)

![image](https://github.com/twilio-labs/paste/assets/9394413/05a157a6-bea5-4068-ad41-a6e8926fa756)

- Are there any examples of this on Console/other Twilio sites?
- Is there any patten we can use for the text?  Currently, it says 'New messages!' but I wasn't sure if there are other guidelines.

Any help would be much appreciated thank you!

**Comment by serifluous** [comment, marked as answer]
Hey @tonge495 ! I'd recommend reaching out to @Porlin @waijeeho and Aren on Flex to check if there's a similar pattern.

**Reply by tonge495** [reply]
Thanks @serifluous!  Will do.

**Reply by tonge495** [reply]
Got in touch with Porlin on this, so I can close out this discussion.
